package.json

```diff
@@ -3,11 +3,7 @@
   "version": "0.7.0",
   "description": "Tab Navigation components for React Navigation",
   "main": "dist/index.js",
-  "files": [
```

Comment:
Don't we still need `dist/` under `files/`, since the `main` field refers to `dist/index.js`. Otherwise, I think it'll break Haul and webpack for people who use it with `react-native-web`

Reply:
I think if you leave files blank then when you publish it should include all files

Reply:
ah yes. should we publish all files though? publishing the whole example and `flow_typed` seems wasteful.

Reply:
yeah I agree, let's whitelist what we want to publish like it is now